src/store/profile/index.js

```diff
@@ -44,7 +54,8 @@ export const reducer = handleActions(
     [updateUserData]: (state, { payload: userData }) => ({
       ...state,
       userData,
-    })
+    }),
+    [fetchUserData]: state => ({ ...state })
```

Comment:
This is unnecessary. We do not necessary need to handle those actions in the reducer that only triggers the proper saga function. We need to handle an action inside the reducer if we want to change the actual value of something in the store when the action dispatch.